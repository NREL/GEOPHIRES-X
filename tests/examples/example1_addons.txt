Starting CCUS Credit Value,0.015
Ending CCUS Credit Value, 0.1
CCUS Escalation Start Year, 5
CCUS Escalation Rate Per Year, 0.01
Current Grid CO2 production,0.82
Starting Heat Sale Price,0.0123
Ending Heat Sale Price,0.0359
Heat Escalation Start Year,7
Heat Escalation Rate Per Year,0.01
Starting Electricity Sale Price,0.09
Ending Electricity Sale Price,0.15
Electricity Escalation Start Year,5
Electricity Escalation Rate Per Year,0.012
Annual License Etc,0
Flat License Etc,0
Other Incentives,2.112
Tax Relief Per Year,2.212
One-time Grants Etc,20.212
AddOn Nickname 1, Desalinization
AddOn CAPEX 1, 10
AddOn OPEX 1, 0.1
AddOn Electricity Gained 1, -100
AddOn Heat Gained 1, 0.0
AddOn Profit Gained 1, 0.05
AddOn Nickname 2, Methane Extration
AddOn CAPEX 2, 20
AddOn OPEX 2, 1
AddOn Electricity Gained 2, 26000.0
AddOn Heat Gained 2, 0
AddOn Profit Gained 2, 2.786
AddOn Nickname 3, CO2 point capture from methane buring
AddOn CAPEX 3, 40
AddOn OPEX 3, 0.6
AddOn Electricity Gained 3, 0.0
AddOn Heat Gained 3, 0
AddOn Profit Gained 3, 0
Reservoir Model,1,                        			---Multiple Fractures reservoir model
Reservoir Depth,3,                     				---[km]
Number of Segments,1,                  				---[-]
Gradient 1,50,                          			---[deg.C/km]
Maximum Temperature,400,                  			---[deg.C]
Number of Production Wells,2,            			---[-]
Number of Injection Wells,2,            			---[-]
Production Well Diameter,7,		     			---[inch]
Injection Well Diameter,7,					---[inch]
Ramey Production Wellbore Model,1,       			---
Production Wellbore Temperature Drop,.5,			---[deg.C]
Injection Wellbore Temperature Gain,0,   			---[deg.C]
Production Flow Rate per Well,55,       			---[kg/s]
Fracture Shape,3,                       			---[-] 
Fracture Height,900, 						---[m]
Reservoir Volume Option,3,              			---[-]
Reservoir Volume,1000000000,		 			---[m^3]
Number of Fractures,20,		  				---[-]
Water Loss Fraction,.02,					---[-]
Productivity Index,5,						---[kg/s/bar]
Injectivity Index,5,						---[kg/s/bar]
Injection Temperature,50,		 			---[deg.C]
Maximum Drawdown,1,			  			---[-] no redrilling considered
Reservoir Heat Capacity,1000,		  			---[J/kg/K]
Reservoir Density,2700,			  			---[kg/m^3]
Reservoir Thermal Conductivity,2.7,	  			---[W/m/K]

***SURFACE TECHNICAL PARAMETERS***
**********************************
End-Use Option,1,			  			---[-] Electricity
Economic Model,1,			  			---[-] Fixed Charge Rate Model
Power Plant Type,2,			  			---[-] Supercritcal ORC
Circulation Pump Efficiency,.8,	  				---[-] between .1 and 1
Utilization Factor,.9,			  			---[-] between .1 and 1
Surface Temperature,20,		  				---[deg.C]
Ambient Temperature,20,		 				---[deg.C]

***FINANCIAL PARAMETERS***
**************************
Plant Lifetime,30,			  			---[years]
Fixed Charge Rate,.05,			 			---[-] between 0 and 1
Inflation Rate During Construction,0,   			---[-] 

***Simulation Parameters***
***************************

Print Output to Console,1,		  			---[-] Should be 0 (don't print results) or 1 (print results)
Time steps per year,6,		  				---[1/year]

