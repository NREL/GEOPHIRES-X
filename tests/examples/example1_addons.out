                               *****************
                               ***CASE REPORT***
                               *****************

Simulation Metadata
----------------------
 GEOPHIRES Version: 3.4.0
 GEOPHIRES Build Date: 2022-06-30
 Simulation Date: 2024-02-12
 Simulation Time:  10:56
 Calculation Time:      0.578 sec

                           ***SUMMARY OF RESULTS***

      End-Use Option: Electricity
      Average Net Electricity Production:                     5.41 MW
      Electricity breakeven price:                            1.74 cents/kWh
      Number of production wells:                             2
      Number of injection wells:                              2
      Flowrate per production well:                          55.0 kg/sec
      Well depth (or total length, if not vertical):          3.0 kilometer
      Geothermal gradient:                                    0.0500 degC/m


                           ***ECONOMIC PARAMETERS***

      Economic Model = Fixed Charge Rate (FCR)
      Fixed Charge Rate (FCR):                                5.00
      Accrued financing during construction:                  0.00
      Project lifetime:                                      30 yr
      Capacity factor:                                       90.0 %
      Project NPV:                                          49.70 MUSD
      Project IRR:                                           0.17 %
      Project VIR=PI=PIR:                                    2.60
      Project MOIC:                                         24.83

                          ***ENGINEERING PARAMETERS***

      Number of Production Wells:                             2
      Number of Injection Wells:                              2
      Well depth (or total length, if not vertical):          3.0 kilometer
      Water loss rate:                                        2.0
      Pump efficiency:                                       80.0
      Injection temperature:                                 50.0 degC
      Production Wellbore heat transmission calculated with Ramey's model
      Average production well temperature drop:               2.9 degC
      Flowrate per production well:                          55.0 kg/sec
      Injection well casing ID:                               7.000 in
      Production well casing ID:                              7.000 in
      Number of times redrilling:                             0
      Power plant type:                                       Supercritical ORC


                         ***RESOURCE CHARACTERISTICS***

      Maximum reservoir temperature:                        400.0 degC
      Number of segments:                                     1
      Geothermal gradient:                                    0.0500 degC/m


                           ***RESERVOIR PARAMETERS***

      Reservoir Model = Multiple Parallel Fractures Model
      Bottom-hole temperature:                              170.00 degC
      Fracture model = Square
      Well seperation: fracture height:                     900.00 meter
      Fracture area:                                     810000.00 m**2
      Reservoir volume:                              1000000000 m**3
      Reservoir hydrostatic pressure:                       100.00 kPa
      Plant outlet pressure:                               1067.94 kPa
      Production wellhead pressure:                        1136.89 kPa
      Productivity Index:                                     5.00 kg/sec/bar
      Injectivity Index:                                      5.00 kg/sec/bar
      Reservoir density:                                   2700.00 kg/m**3
      Reservoir thermal conductivity:                         2.70 watt/m/kelvin
      Reservoir heat capacity:                             1000.00 J/kg/kelvin


                           ***RESERVOIR SIMULATION RESULTS***

      Maximum Production Temperature:                       167.3 degC
      Average Production Temperature:                       167.1 degC
      Minimum Production Temperature:                       165.4 degC
      Initial Production Temperature:                       165.4 degC
      Average Reservoir Heat Extraction:                     54.35 MW
      Production Wellbore Heat Transmission Model = Ramey Model
      Average Production Well Temperature Drop:               2.9 degC
      Average Injection Well Pump Pressure Drop:            217.9 kPa
      Average Production Well Pump Pressure Drop:          1004.5 kPa


                          ***CAPITAL COSTS (M$)***

         Drilling and completion costs:                      21.95 MUSD
         Drilling and completion costs per well:              5.49 MUSD
         Stimulation costs:                                   3.02 MUSD
         Surface power plant costs:                          20.81 MUSD
         Field gathering system costs:                        2.29 MUSD
         Total surface equipment costs:                      23.10 MUSD
         Exploration costs:                                   5.33 MUSD
      Total capital costs:                                   31.07 MUSD
      Annualized capital costs:                               1.55 MUSD


                ***OPERATING AND MAINTENANCE COSTS (M$/yr)***

         Wellfield maintenance costs:                         0.44 MUSD/yr
         Power plant maintenance costs:                       0.90 MUSD/yr
         Water costs:                                         0.06 MUSD/yr
      Total operating and maintenance costs:                 -0.82 MUSD/yr


                           ***SURFACE EQUIPMENT SIMULATION RESULTS***

      Initial geofluid availability:                          0.11 MW/(kg/s)
      Maximum Total Electricity Generation:                   5.62 MW
      Average Total Electricity Generation:                   5.59 MW
      Minimum Total Electricity Generation:                   5.43 MW
      Initial Total Electricity Generation:                   5.43 MW
      Maximum Net Electricity Generation:                     5.44 MW
      Average Net Electricity Generation:                     5.41 MW
      Minimum Net Electricity Generation:                     5.24 MW
      Initial Net Electricity Generation:                     5.24 MW
      Average Annual Total Electricity Generation:           43.89 GWh
      Average Annual Net Electricity Generation:             42.44 GWh
      Initial pumping power/net installed power:            100.00 %
      Average Pumping Power:                                  0.18 MW

                            ************************************************************
                            *  HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                            ************************************************************
  YEAR       THERMAL               GEOFLUID               PUMP               NET               FIRST LAW
             DRAWDOWN             TEMPERATURE             POWER             POWER              EFFICIENCY
                                     (degC)               (MW)              (MW)                  (%)
   1           1.0000                165.41               0.1865            5.2425                9.7847
   2           1.0056                166.33               0.1855            5.3341                9.8771
   3           1.0071                166.59               0.1852            5.3605                9.9036
   4           1.0080                166.73               0.1851            5.3743                9.9174
   5           1.0085                166.82               0.1850            5.3835                9.9265
   6           1.0089                166.89               0.1849            5.3902                9.9332
   7           1.0093                166.94               0.1848            5.3955                9.9385
   8           1.0095                166.98               0.1848            5.3999                9.9428
   9           1.0097                167.02               0.1848            5.4036                9.9465
  10           1.0099                167.05               0.1847            5.4067                9.9497
  11           1.0101                167.08               0.1847            5.4095                9.9524
  12           1.0102                167.10               0.1847            5.4120                9.9549
  13           1.0104                167.12               0.1846            5.4142                9.9571
  14           1.0105                167.14               0.1846            5.4162                9.9591
  15           1.0106                167.16               0.1846            5.4180                9.9609
  16           1.0107                167.18               0.1846            5.4197                9.9626
  17           1.0108                167.20               0.1846            5.4213                9.9641
  18           1.0109                167.21               0.1846            5.4227                9.9656
  19           1.0110                167.22               0.1845            5.4241                9.9669
  20           1.0110                167.24               0.1845            5.4254                9.9682
  21           1.0111                167.25               0.1845            5.4266                9.9694
  22           1.0112                167.26               0.1845            5.4277                9.9705
  23           1.0113                167.27               0.1845            5.4288                9.9716
  24           1.0113                167.28               0.1845            5.4298                9.9726
  25           1.0114                167.29               0.1845            5.4307                9.9735
  26           1.0114                167.30               0.1845            5.4316                9.9744
  27           1.0115                167.31               0.1845            5.4325                9.9753
  28           1.0115                167.31               0.1844            5.4333                9.9761
  29           1.0116                167.32               0.1844            5.4341                9.9768
  30           1.0116                167.33               0.1844            5.4348                9.9776


                              *******************************************************************
                              *  ANNUAL HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                              *******************************************************************
  YEAR             ELECTRICITY                   HEAT                RESERVOIR            PERCENTAGE OF
                    PROVIDED                   EXTRACTED            HEAT CONTENT        TOTAL HEAT MINED
                   (GWh/year)                  (GWh/year)            (10^15 J)                 (%)
   1                  41.8                       424.3                322.47                   0.47
   2                  42.2                       426.3                320.94                   0.95
   3                  42.3                       427.0                319.40                   1.42
   4                  42.4                       427.4                317.86                   1.89
   5                  42.5                       427.7                316.32                   2.37
   6                  42.5                       427.9                314.78                   2.85
   7                  42.6                       428.1                313.24                   3.32
   8                  42.6                       428.2                311.70                   3.80
   9                  42.6                       428.4                310.16                   4.27
  10                  42.7                       428.5                308.61                   4.75
  11                  42.7                       428.6                307.07                   5.22
  12                  42.7                       428.7                305.53                   5.70
  13                  42.7                       428.7                303.98                   6.18
  14                  42.7                       428.8                302.44                   6.65
  15                  42.7                       428.9                300.90                   7.13
  16                  42.8                       428.9                299.35                   7.61
  17                  42.8                       429.0                297.81                   8.08
  18                  42.8                       429.0                296.26                   8.56
  19                  42.8                       429.1                294.72                   9.04
  20                  42.8                       429.1                293.17                   9.51
  21                  42.8                       429.2                291.63                   9.99
  22                  42.8                       429.2                290.08                  10.47
  23                  42.8                       429.2                288.54                  10.94
  24                  42.8                       429.3                286.99                  11.42
  25                  42.8                       429.3                285.45                  11.90
  26                  42.9                       429.3                283.90                  12.38
  27                  42.9                       429.4                282.36                  12.85
  28                  42.9                       429.4                280.81                  13.33
  29                  42.9                       429.4                279.27                  13.81
  30                  35.7                       357.9                277.98                  14.20



                                ***EXTENDED ECONOMICS***

      Adjusted Project LCOE (after incentives, grants, AddOns,etc):           1.74 cents/kWh
      Adjusted Project LCOH (after incentives, grants, AddOns,etc):           0.00 USD/MMBTU
      Adjusted Project CAPEX (after incentives, grants, AddOns, etc):       101.07 MUSD
      Adjusted Project OPEX (after incentives, grants, AddOns, etc):          0.88 MUSD
      Project NPV   (including AddOns):                                      -4.98 MUSD
      Project IRR   (including AddOns):                                       0.06 %
      Project VIR=PI=PIR   (including AddOns):                                0.95
      Project MOIC  (including AddOns):                                       1.00
      Project Payback Period       (including AddOns):                       15.57 yr
      Total Add-on CAPEX:                                                    70.00 MUSD
      Total Add-on OPEX:                                                      1.70 MUSD/yr
      Total Add-on Net Elec:                                              25900.00 kW/yr
      Total Add-on Net Heat:                                                  0.00 kW/yr
      Total Add-on Profit:                                                    2.84 MUSD/yr
      AddOns Payback Period:                                                  0.00 yr


                             *******************************
                             *  EXTENDED ECONOMIC PROFILE  *
                             *******************************
Year        Electricity             Heat             Add-on  Annual AddOn Cumm. AddOn  Annual Project Cumm. Project
Since     Price   Revenue      Price   Revenue      Revenue   Cash Flow    Cash Flow    Cash Flow       Cash Flow
Start    (cents/kWh)(MUSD/yr) (cents/kWh)(MUSD/yr) (MUSD/yr)    (MUSD/yr)  (MUSD)       (MUSD/yr)        (MUSD)
     1    0.090   0.0023        0.012   0.0000         1.14        -70.00     -70.00        -101.07           -101.07
     2    0.090   0.0023        0.012   0.0000         1.14         1.14     -68.86         5.71           -95.36
     3    0.090   0.0023        0.012   0.0000         1.14         1.14     -67.72         5.75           -89.61
     4    0.090   0.0023        0.012   0.0000         1.14         1.14     -66.59         5.77           -83.84
     5    0.090   0.0023        0.012   0.0000         1.14         1.14     -65.45         5.77           -78.07
     6    0.090   0.0023        0.012   0.0000         1.14         1.14     -64.31         5.78           -72.29
     7    0.102   0.0026        0.012   0.0000         1.14         1.14     -63.17         5.78           -66.51
     8    0.114   0.0030        0.012   0.0000         1.14         1.14     -62.03         6.30           -60.21
     9    0.126   0.0033        0.022   0.0000         1.14         1.14     -60.89         6.81           -53.40
    10    0.138   0.0036        0.032   0.0000         1.14         1.14     -59.75         7.33           -46.07
    11    0.150   0.0039        0.036   0.0000         1.14         1.14     -58.61         7.84           -38.23
    12    0.150   0.0039        0.036   0.0000         1.14         1.14     -57.47         8.36           -29.87
    13    0.150   0.0039        0.036   0.0000         1.14         1.14     -56.33         8.36           -21.51
    14    0.150   0.0039        0.036   0.0000         1.14         1.14     -55.19         8.36           -13.14
    15    0.150   0.0039        0.036   0.0000         1.14         1.14     -54.05         8.37           -4.78
    16    0.150   0.0039        0.036   0.0000         1.14         1.14     -52.91         8.37            3.59
    17    0.150   0.0039        0.036   0.0000         1.14         1.14     -51.77         8.37           11.96
    18    0.150   0.0039        0.036   0.0000         1.14         1.14     -50.63         8.37           20.33
    19    0.150   0.0039        0.036   0.0000         1.14         1.14     -49.49         8.37           28.70
    20    0.150   0.0039        0.036   0.0000         1.14         1.14     -48.35         8.37           37.08
    21    0.150   0.0039        0.036   0.0000         1.14         1.14     -47.21         8.38           45.46
    22    0.150   0.0039        0.036   0.0000         1.14         1.14     -46.07         8.38           53.83
    23    0.150   0.0039        0.036   0.0000         1.14         1.14     -44.93         8.38           62.21
    24    0.150   0.0039        0.036   0.0000         1.14         1.14     -43.80         8.38           70.59
    25    0.150   0.0039        0.036   0.0000         1.14         1.14     -42.66         8.38           78.97
    26    0.150   0.0039        0.036   0.0000         1.14         1.14     -41.52         8.38           87.36
    27    0.150   0.0039        0.036   0.0000         1.14         1.14     -40.38         8.38           95.74
    28    0.150   0.0039        0.036   0.0000         1.14         1.14     -39.24         8.38           104.12
    29    0.150   0.0039        0.036   0.0000         1.14         1.14     -38.10         8.39           112.51
    30    0.150   0.0039        0.036   0.0000         1.14         1.14     -36.96         8.39           120.89


                                ***CCUS ECONOMICS***


      Total Avoided Carbon Production:                       1043955409.30 pound
      Project NPV            (including carbon credit):           75.61 MUSD
      Project IRR            (including carbon credit):            0.20 %
      Project VIR=IR=PIR     (including carbon credit):            3.43
      Project MOIC           (including carbon credit):           36.29
      Project Payback Period (including carbon credit):            7.04 yr


                    ******************
                    *  CCUS PROFILE  *
                    ******************
Year       Carbon          CCUS               CCUS Annual   CCUS Cumm.  Project Annual   Project Cumm.
Since      Avoided    Price   Revenue          Cash Flow    Cash Flow      Cash Flow       Cash Flow
Start      (pound)   (USD/lb) (MUSD/yr)        (MUSD/yr)    (MUSD)         (MUSD/yr)        (MUSD)
     1                                                                     -31.07           -31.07
     2  34242035.250  0.015    0.51              0.51         0.51           5.09           -25.98
     3  34602531.668  0.015    0.52              0.52         1.03           5.13           -20.85
     4  34724553.252  0.015    0.52              0.52         1.55           5.15           -15.70
     5  34796819.098  0.015    0.52              0.52         2.08           5.16           -10.55
     6  34847475.212  0.015    0.52              0.52         2.60           5.16           -5.38
     7  34886120.252  0.015    0.52              0.52         3.12           5.17           -0.21
     8  34917165.394  0.025    0.87              0.87         3.99           6.03            5.82
     9  34942993.055  0.035    1.22              1.22         5.22           6.90           12.71
    10  34965030.710  0.045    1.57              1.57         6.79           7.76           20.48
    11  34984198.487  0.055    1.92              1.92         8.71           8.63           29.10
    12  35001122.677  0.065    2.28              2.28        10.99           9.49           38.60
    13  35016247.966  0.075    2.63              2.63        13.62           9.85           48.44
    14  35029900.940  0.085    2.98              2.98        16.59          10.20           58.64
    15  35042328.175  0.095    3.33              3.33        19.92          10.55           69.20
    16  35053720.175  0.100    3.51              3.51        23.43          10.73           79.93
    17  35064226.998  0.100    3.51              3.51        26.93          10.74           90.67
    18  35073968.794  0.100    3.51              3.51        30.44          10.74           101.41
    19  35083043.073  0.100    3.51              3.51        33.95          10.74           112.15
    20  35091529.833  0.100    3.51              3.51        37.46          10.74           122.89
    21  35099495.213  0.100    3.51              3.51        40.97          10.75           133.64
    22  35106994.126  0.100    3.51              3.51        44.48          10.75           144.39
    23  35114072.168  0.100    3.51              3.51        47.99          10.75           155.14
    24  35120766.996  0.100    3.51              3.51        51.50          10.75           165.89
    25  35127109.346  0.100    3.51              3.51        55.02          10.75           176.64
    26  35133123.761  0.100    3.51              3.51        58.53          10.76           187.40
    27  35138829.140  0.100    3.51              3.51        62.04          10.76           198.16
    28  35144239.144  0.100    3.51              3.51        65.56          10.76           208.92
    29  35149362.497  0.100    3.51              3.51        69.07          10.76           219.68
    30  35154203.228  0.100    3.52              3.52        72.59          10.76           230.44
    31  29302202.672  0.100    2.93              2.93        75.52           9.11           239.54
